Code.gs

```diff
@@ -156,6 +170,7 @@ function main(){
   //----------------------------------------------------------------
   
   if(addEventsToCalendar || removeEventsFromCalendar){
+    /* We might want to consider reducing how far back we reach for events to reduce API call usage... Also would make a good premium feature */
```

Comment:
Good point, but let's take this comment out so we don't have comments about "possible premium features" in our code :)